api/services/handle_event_service.py

```diff
@@ -23,6 +25,14 @@ def handle_event_service(self, form_data):
         if not system_phone_exists:
             return
 
+        user_details_exist = form_data.get("From")
+
+        if user_details_exist is None:
+            logger.warning(
+                f"User details are missing in the webhook payload: {form_data}"
+            )
+            return
+

```

Comment:
Here we are checking if the user details are present in the webhook we are getting from the KooKoo. If not then we are skipping those webhooks to be processed.